tests/perf_v2/tasks/detection.py

```diff
@@ -21,6 +21,16 @@
 MODEL_TEST_CASES = [
     ModelInfo(task=TASK_TYPE.value, name="atss_mobilenetv2", category="default"),
     ModelInfo(task=TASK_TYPE.value, name="yolox_s", category="speed"),
+    ModelInfo(task=TASK_TYPE.value, name="dfine_x", category="accuracy"),
```

Comment:
It may be not directly related to this PR, but it would be more readable if `ModelInfo` can accept both `str` and `TASK_TYPE`, so a user would not need to pass `.value` every time.

It would overall become something like 
```python
ModelInfo(task=TASK_TYPE, name="dfine_x", category="accuracy"),
```